test/dotnet-migrate.Tests/GivenThatIWantToMigrateTestApps.cs

```diff
@@ -101,6 +112,16 @@ public void It_migrates_dotnet_new_web_with_outputs_containing_project_json_outp
             outputsIdentical.Should().BeTrue();
         }
 
+        private void WriteGlobalJson(string globalDirectory)
+        {
+            var file = Path.Combine(globalDirectory, "global.json");
+
+            File.WriteAllText(file, @"
+            {
+                ""projects"": [ ]
+            }");
+        }
+
         [Theory]
         // TODO: Enable this when X-Targeting is in
```

Comment:
No I saw this too, turned it back on but it's failing, going to fix it today and make a separate PR. There's another test in this file that needs to be turned on too